rtl/include/eclhelper.hpp

```diff
@@ -48,8 +48,8 @@ typedef unsigned short UChar;
 
 //Should be incremented whenever the virtuals in the context or a helper are changed, so
 //that a work unit can't be rerun.  Try as hard as possible to retain compatibility.
-#define ACTIVITY_INTERFACE_VERSION      653
-#define MIN_ACTIVITY_INTERFACE_VERSION  650             //minimum value that is compatible with current interface
+#define ACTIVITY_INTERFACE_VERSION      654
+#define MIN_ACTIVITY_INTERFACE_VERSION  651             //minimum value that is compatible with current interface
```

Comment:
You shouldn't need to increment the min value (if you did it would need to be 654).  Using a global function would mean this didn't need to be changed.